### ISSUE TITLE
Unable to activate plugin

### ISSUE THREAD

**Issue by hcayless** [Conal-Tuohy/jEdit-TEIPlugin#15]
I'm getting the following error after installing the TEI plugin. Possibly related to Saxon API updates? I've been bitten by this sort of thing in other contexts and this looks familiar. Saxon seems to change its API every new moon.

```
/Users/hac13/Library/jEdit/jars/TEI.jar:
Cannot start: java.lang.NoSuchMethodError: net.sf.saxon.Configuration.setProcessor(Lnet/sf/saxon/Configuration$ApiProvider;)V
Try updating to a newer version of the plugin.
```

**Comment by Conal-Tuohy** [comment]
Thanks! What version of the Saxon jEdit plugin is installed? And could you also check the version of the Saxon jar in your jedit jars folder?

**Comment by hcayless** [comment]
Saxon HE 9.9.1.6 (plugin version number is the same—I guess it tracks the Saxon version). That's not quite cutting edge, but recent. As I look back on how I handled changes to the Configuration API previously, I see I switched to using [s9api](http://saxonica.com/html/documentation/configuration/config-interfaces/s9api-config.html) (can't remember why now, but maybe it seemed more stable). I see they're retaining the name for Saxon 10, so that's good.

**Comment by hcayless** [comment]
But there is indeed a [Configuration.setProcessor](http://saxonica.com/html/documentation9.9/javadoc/net/sf/saxon/Configuration.html#setProcessor-net.sf.saxon.Configuration.ApiProvider-) with the right signature, so your code is correct. Looking a little deeper, I found a `saxon7.jar` stuck in my jars/ folder, left over from God knows when. Because these things are loaded in alphanumeric order, it will have been stepping on the newer version. I killed it and the TEI plugin loads now. I'll report back if I find any issues with it!
